diff --git a/node_modules/@ef-carbon/tspm/dist/bin/tspm b/node_modules/@ef-carbon/tspm/dist/bin/tspm
index c9e70df..a66243f 100644
--- a/node_modules/@ef-carbon/tspm/dist/bin/tspm
+++ b/node_modules/@ef-carbon/tspm/dist/bin/tspm
@@ -169,6 +169,7 @@ class Declaration {
         const { resolvedFileName: pathWithExtension, extension, isExternalLibraryImport: external } = resolved;
         const path$$1 = path.join(path.dirname(pathWithExtension), path.basename(pathWithExtension, extension));
         let resolution = path.relative(this.file.source.parent.toString(), path$$1);
+        resolution = resolution.replace('\\', '/')
         if (!resolution.startsWith('.')) {
             resolution = `./${resolution}`;
         }
